.github/workflows/actions.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [2.6.8, 2.5.9]
+        ruby-version: [2.6.8]
```

Comment:
Could you add `2.7.5` (ZAM) to this matrix? :)

Reply:
Thinking about that, should we replace `2.6.8` with `2.6.10` and add `3.0.4` and `3.1.2` as well?